app/models/job/JobService.scala

```diff
@@ -75,6 +74,11 @@ class JobService @Inject()(wkConf: WkConf,
     ()
   }
 
+  private def durationLabel(job: Job) = {
+    val waitLabel = job.waitDuration.map(w => s"waiting ${formatDuration(w)} and running ").getOrElse("")
```

Comment:
This reads like the job is still running. But what if the job already finished? Then the label would still say "and running " :thinking: